lib/gooddata/lcm/actions/synchronize_user_groups.rb

```diff
@@ -27,31 +27,44 @@ class SynchronizeUserGroups < BaseAction
 
         description 'Additional Hidden Parameters'
         param :additional_hidden_params, instance_of(Type::HashType), required: false
+
+        description 'Disable dashboard permission for pixel perfect dashboard'
+        param :disable_pp_dashboard_permission, instance_of(Type::BooleanType), required: false, default: false
+
+        description 'Disable dashboard permission for KD dashboard'
+        param :disable_kd_dashboard_permission, instance_of(Type::BooleanType), required: false, default: false
       end
 
       class << self
         def call(params)
           results = ThreadSafe::Array.new
 
-          client = params.gdc_gd_client
-          development_client = params.development_client
+          disable_pp_dashboard_permission = GoodData::Helpers.to_boolean(params.disable_pp_dashboard_permission)
+          disable_kd_dashboard_permission = GoodData::Helpers.to_boolean(params.disable_kd_dashboard_permission)
+
+          if disable_pp_dashboard_permission && disable_kd_dashboard_permission
```

Comment:
User groups is always synchronized before, we shouldn't change this behaviour, no need to check params disable_pp_dashboard_permission && disable_kd_dashboard_permission